build.rs

```diff
@@ -518,13 +516,15 @@ fn windows_hardcoded_cross_compile(
 fn load_cross_compile_info(
     cross_compile_config: CrossCompileConfig,
 ) -> Result<(InterpreterConfig, BuildFlags)> {
-    let target_family = env::var("CARGO_CFG_TARGET_FAMILY")?;
-    // Because compiling for windows on linux still includes the unix target family
-    if target_family == "unix" {
+    match env::var("CARGO_CFG_TARGET_FAMILY") {
```

Comment:
Hmm the `Err` can include utf8 decoding errors too. We can avoid those using `var_os`:

```suggestion
    match env::var_os("CARGO_CFG_TARGET_FAMILY") {
```